src/models/apis/googleMaps.ts

```diff
@@ -31,7 +31,7 @@ export default class GoogleMaps implements IMapFunctions {
 
     constructor() { /* */ }
 
-    public initialize(mapType: MapType, params: any, elementId: string) {
+    public initialize(mapType: MapType, params: any, elementId?: string): Promise<any> {
```

Comment:
no lugar de nulable para elementId, não daria pra usar um default value?
```suggestion
    public initialize(mapType: MapType, params: any, elementId= 'inlog-map'): Promise<any> {
```

Reply:
Alteração adicionada